README.md

```diff
@@ -41,16 +43,44 @@ mkstage4 -t /custom/mount/point archive_name
 Command line arguments:
 
 ```
-mkstage4 [-q -c -b] [-s || -t <target-mountpoint>] <archive-filename> [custom-tar-options]
+  mkstage4.sh [-q -c -b -l -k] [-s || -t <target-mountpoint>] [-e <additional excludes dir*>] <archive-filename> [custom-tar-options]
   -q: activates quiet mode (no confirmation).
   -c: excludes connman network lists.
   -b: excludes boot directory.
   -l: excludes lost+found directory.
+  -e: an additional excludes directory (one dir one -e).
   -s: makes tarball of current system.
+  -k: separately save current kernel modules and src (smaller & save decompression time).
   -t: makes tarball of system located at the <target-mountpoint>.
   -h: displays help message.
 ```
 
+eg.
```

Comment:
Please post only the command instead of the entire terminal output (will make the examples more sustainable as output may change independently of usage).

Also, please place the example under the “Archive system located at a custom mount point:” example, and formatted analogously.